src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -461,16 +480,42 @@
     }
   }
 
+  .topRowWrapper {
+    display: flex;
+    flex-direction: row;
+    gap: min(var(--space-lg), 50px);
```

Comment:
this gap is not really needed, we have only 2 children and they are separated on the 2 sides of the element

Reply:
Fixed